proposals/lip-0038.md

```diff
@@ -153,12 +159,16 @@ We choose the variant _[minimal-pubkey-size](https://tools.ietf.org/html/draft-i
 
 #### Choice of Scheme
 
-We use the _proof of possession_ scheme as we only need the use case of aggregating signatures of the same message, and this scheme allows us to use `FastAggregateVerify` for this case. `FastAggregateVerify`requires only two pairing operations whereas `AggregateVerify` requires `n`+1 pairing operations where `n` is the number of individual signatures that are aggregated. Note that pairing operations are very expensive.
+We use the _proof of possession_ scheme as we only need the use case of aggregating signatures of the same message, and this scheme allows us to use `FastAggregateVerify` for this case. `FastAggregateVerify` requires only two pairing operations whereas `AggregateVerify` requires `n`+1 pairing operations where `n` is the number of individual signatures that are aggregated. Note that pairing operations are very expensive.
 
 #### Why Proofs of Possession?
 
 Using `FastAggregateVerify` without requiring proofs of possession is insecure as it allows rogue key attacks. See this [blog post](https://medium.com/@coolcottontail/rogue-key-attack-in-bls-signature-and-harmony-security-eac1ea2370ee) for how rogue key attacks work for BLS signatures. To see why simply signing the public key is not a sufficient proof of possession method that defends against powerful attackers (chosen message attack model), see section 4.3 of [this paper](https://www.iacr.org/archive/eurocrypt2007/45150228/45150228.pdf).
 
+### Why Additional Rule for FastAggregateVerify?
+
+Lisk Core v4 poses an exception to the rule that only public keys, for which a valid proof of possession exists, can be used. More precisely, during the starting phase of Lisk Core v4, there will be validators with the invalid BLS public key where all 48 bytes are set to zero. See [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#assigning-bls-key-to-validator-accounts) for more details why this is needed. Due to this, it can happen that Lisk Core v4 nodes or even sidechain nodes must evaluate `FastAggregateVerify` where one of the input public keys is the one where all 48 bytes are set to zero. Although the IETF draft does not specify the behavior, the used BLS-library in the Lisk SDK 6 does actually return `INVALID/False` for this specific case. Therefore, it was considered easier to add this extra rule that matches with the implementation to this LIP instead of adding additional rules in the SDK that prevent that `FastAggregateVerify` is called with the key where all 48 bytes are set to zero.
```

Comment:
```suggestion
Lisk Core v4 poses an exception to the rule that only public keys for which a valid proof of possession exists can be used. More precisely, during the starting phase of Lisk Core v4, there will be validators with the invalid BLS public key where all 48 bytes are set to zero. See [here](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0063.md#assigning-bls-key-to-validator-accounts) for more details why this is needed. Due to this, it can happen that Lisk Core v4 nodes or even sidechain nodes must evaluate `FastAggregateVerify` where one of the input public keys is the one where all 48 bytes are set to zero. Although the IETF draft does not specify the behavior, the used BLS-library in the Lisk SDK 6 does actually return `INVALID/False` for this specific case. Therefore, it was considered easier to add this extra rule that matches with the implementation to this LIP instead of adding additional rules in the SDK that prevent that `FastAggregateVerify` is called with the key where all 48 bytes are set to zero.
```

Reply:
Somehow flows better to me, feel free to disregard.